docs/reference/items.md

```diff
@@ -413,81 +575,132 @@ mutation {
 	}
 }
 ```
+</template>
+</SnippetToggler>
+
 
 ---
 
 ## Delete an Item
 
 Delete an existing item.
 
-### Returns
+### Request
 
-Empty body.
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
 
-### REST API
+<template #rest>
 
-```
-DELETE /items/:collection/:id
-```
-
-##### Example
-
-```
-DELETE /items/articles/15
-```
+`DELETE /items/:collection/:id`
+</template>
 
-### GraphQL
-
-```
-POST /graphql
-```
+<template #graphql>
 
+`POST /graphql`
 ```graphql
 type Mutation {
 	delete_<collection>_item(id: ID!): delete_one
 }
 ```
+</template>
+</SnippetToggler>
+
+### Response
 
-##### Example
+Empty body.
+
+### Example
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+
+<template #rest>
+
+`DELETE /items/articles/15`
+</template>
 
+<template #graphql>
+
+`POST /graphql`
 ```graphql
 mutation {
 	delete_articles_item(id: 15) {
 		id
 	}
 }
 ```
+</template>
+
+</SnippetToggler>
 
 ---
 
 ## Delete Multiple Items
 
 Delete multiple existing items.
 
-### Query Parameters
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+
+<template #rest>
+
+`DELETE /items/:collection`
+```json
+// Array
+[value_1, value_2, value_3]
+```
+
+```json
+// Object 
+{
+	"field": [value_1, value_2, value_3]
+}
+```
+
+```json
+// Object containing query
+{
+	"query": {
+		"query_type": {
```

Comment:
@phazonoverload was working backwards from the example here and using filter docs to see what you call the generics but I dont know if got it right